hack/build.sh

```diff
@@ -13,6 +13,6 @@ go mod tidy
 go build $repro_flags -mod=vendor -o controller ./cmd/main.go
 ls -la ./controller
 
-ytt -f config/ >/dev/null
+ytt -f config/ --file-mark 'crd-overlay.yml:exclude=true' >/dev/null
```

Comment:
why do we need to exclude this file? is it because it's used only during generation of crds.yml? if so, i think we should move it to a different location (like hack directory next to the script that it uses it?)

Reply:
Yes, mentioned in comment above. As of now, overlay is only used for supplementing the generation of crds. I have moved to hack folder for now and removed file-marks. Let me know if there is a more logical place for it though.